 Handle IllegalArgumentException

In some cases startInputInnerLocked() throws
IllegalArgumentException which needs to be handle

diff --git a/services/core/java/com/android/server/InputMethodManagerService.java b/services/core/java/com/android/server/InputMethodManagerService.java
index a16fcd2..2741152 100644
--- a/services/core/java/com/android/server/InputMethodManagerService.java
+++ b/services/core/java/com/android/server/InputMethodManagerService.java
@@ -1480,7 +1480,12 @@ InputBindResult startInputUncheckedLocked(@NonNull ClientState cs, IInputContext
             }
         }
 
-        return startInputInnerLocked();
+        try {
+            return startInputInnerLocked();
+        } catch (RuntimeException e) {
+            Slog.w(TAG, "Unexpected exception", e);
+            return null;
+        }
     }
 
     InputBindResult startInputInnerLocked() {
